modules/configuration-ovnk-network-plugin-json-object.adoc

```diff
@@ -66,4 +66,9 @@ A comma-separated list of CIDRs and IP addresses. IP addresses are removed from
 |
 If topology is set to `localnet`, the specified VLAN tag is assigned to traffic from this additional network. The default is to not assign a VLAN tag.
 
+|`physicalNetworkName`
+|`string`
+|
+The name of the physical network to which the OVN overlay connects. When omitted, the default value is the `name` of the `localnet` network. By using this field, you can reuse the same physical network mapping with multiple network overlays. Ensure that a different `vlanID` is used when sharing the same physical network between overlays.
```

Comment:
The user may actually want to "mix" the networks ... Not sure if that's a use case or not.

But this could go the other way around, something like:
```suggestion
The name of the physical network to which the OVN overlay connects. When omitted, the default value is the `name` of the `localnet` network. By using this field, you can reuse the same physical network mapping with multiple network overlays. Consider using a different `vlanID` when sharing the same physical network between overlays in order to keep the networks isolated, if desired.
```

I'm not sure it reads better. Maybe better to get feedback from stake-holders.

@phoracek / @jcaamano do you have a strong opinion here ?